### PR TITLE
Sprint58:  mail contents for the H-Branding fee past due alert email

### PR DESCRIPTION
**Pull request by weedySeaDragon**
### PT Story:  Email alert: to all in a company: H-branding is not paid
https://www.pivotaltracker.com/story/show/161889566

This PR is open so that the email _contents_ can be specified.
@thesuss -- This PT story is all ready except for the contents of the emails.  You can update this PR yourself or let me know what the text should be.

### Changes proposed in this pull request:
1.  add preview method for the H-Brand fee past due email

### Screenshots

hbrand past due email sv 2019-01-03 113503

---

hbrand past due email sv text 2019-01-03 113517

---

hbrand past due email en 2019-01-03 113433

---

Ready for review:
@thesuss

### CODE REVIEW

**Comment by thesuss** [comment]
I dont see the suggested text? 

Here's a suggestion where I just changes 4 words. 

Hej FULLNAME

Observera att **ditt företags** nuvarande **H-märkespremie** löper ut DATE.

Du kan förlänga **den** i ytterligare 12 månader genom att betala.

Mitt **företag** (du kan behöva logga in först innan länken fungerar).

**Comment by weedySeaDragon** [comment]
@thesuss  the view file for that mail is what needs to be completed.

 https://github.com/AgileVentures/shf-project/blob/01d35c65dbd6818fd33d3060f0d15f1f2a1aa0fe/app/views/member_mailer/h_branding_fee_past_due.html.haml

I think what I sketched out is pretty close to what you have.  (But if not, fix it).
As usual, you need to make sure the local file entries are correct.

File: config/locales/en.yml [hunk 1]
```diff
@@ -1035,6 +1035,14 @@ en:
                              12 months by paying the membership fee.
           account_login_html: "%{account_link} (you may need to login first before the link works)."
 
+      h_branding_fee_past_due:
+        subject: The H-Branding fee for your company is unpaid/past due
+        greeting: *email_greeting
+        message_text:
+          fee_is_unpaid: "The branding fee for your company %{company_name} has not yet been paid."
+          how_to_pay_fee: "To pay the fee, go to the xxx page and ..."
+          account_login_html: "%{account_link} (you may need to login first before the link works)."
```
Comment (thesuss) [inline config/locales/en.yml]: It needs to be company page login

           fee_is_unpaid: Please note that your current H-branding fee is unpaid.
          how_to_pay_fee: You can pay your H-branding for another
                             12 months by paying the fee. This makes you visible on the map. 
          company_login_html: "%{company_link} (you may need to login first before you can see the button to pay)."

File: config/locales/sv.yml [hunk 2]
```diff
@@ -1037,6 +1037,14 @@ sv:
                                   12 månader genom att betala medlemsavgiften.
           account_login_html: "%{account_link} (du kan behöva logga in först innan länken fungerar)."
 
+      h_branding_fee_past_due:
+        subject: The H-Branding fee for your company is unpaid/past due
+        greeting: *email_greeting
+        message_text:
+          fee_is_unpaid: "The branding fee for your company %{company_name} has not yet been paid."
+          how_to_pay_fee: "To pay the fee, go to the xxx page and ..."
+          account_login_html: "%{account_link} (you may need to login first before the link works)."
```
Comment (thesuss) [inline config/locales/sv.yml]: It needs to be company page login

`subject: Ditt företags H-märkespremie är obetald. `
 `fee_is_unpaid: Observera att ditt företags H-märkespremie inte är betald. `
`how_to_pay_fee: Betala den ett år till via länken nedan. Detta gör att du fortsätter synas på kartan och fortsätter ha ett H-märkt företag. `
`company_login_html: "%{company_link} (du kan behöva logga in först innan du ser knappen för att betala).`

**Review by thesuss — CHANGES_REQUESTED** [review]

**Comment by thesuss** [comment]
I think its good, can you make a picture to make sure the link is in there? :)

**Comment by weedySeaDragon** [comment]
@thesuss : I added the screenshots to the PR description.

(Thanks for reminding me to do that.)

**Review by thesuss — APPROVED** [review]

**Pull request merged**